site/layouts/partials/navbar.html

```diff
@@ -239,11 +239,9 @@
   const utmContent = params.get('utm_content')
   if (utmContent) newParams.set('utm_content', utmContent)
 
-  const loginButton = document.querySelector('#login-button')
-  loginButton.href = `https:\/\/app.checklyhq.com/?${newParams.toString()}`
-  const dashboardButton = document.querySelector('#dashboard-button')
-  dashboardButton.href = `https:\/\/app.checklyhq.com/?${newParams.toString()}`
-  const signupButton = document.querySelector('#nav-signup-button')
-  signupButton.href = `https:\/\/app.checklyhq.com/signup?${newParams.toString()}`
+  const signupLinks = document.querySelectorAll('a[href*="app.checklyhq.com"]')
```

Comment:
This is a great first step and covers all occurrences.

Additionally, we'd like to keep track of the entry campaign across site navigations. We want to attribute folks coming in with UTMs even when they're navigating across the site.

E.g. it's unlikely that a person lands on a blog and signs up right away. A more natural behavior would be
some blog with UTMs -> what's this checkly thing `/` -> what's the price `/pricing` -> Cool. Let's give it a try signup

For that then, we'd need to persist the initial `UTMs` and entry data (`localstorage` or `cookie` I guess)

The point is: if someone enters [checklyhq.com](http://checklyhq.com/) with UTMs and signs up weeks later after unknown navigations we'd like to attribute it to the first campaign.

Reply:
Yeah great point. So it's relatively easy to save them on initial visit to `localStorage` and then reapply them on page load, if they exist. 

However, how should we go about cleaning them up?

What we dont want (I assume), is when a user comes back to `checklyhq.com` two weeks later and we reapply the same UTM tags that they had from some visit two weeks prior. Right?

But we can't just delete the `localStorage` items on every `pageUnload`, right? Cause that would just remove them on any navigation around the site whcih is exactly the opposite of what we're trying to achieve haha.

Any good ideas for this?